app/services/dataset_transformer_loader.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# Performs transformation from source metadata to Solr documents and loading into Solr for a single dataset
+class DatasetTransformerLoader
+  # These providers are ordered by preference for mapping.
+  PROVIDERS = %w[sdr datacite local searchworks dryad redivis zenodo].freeze
```

Comment:
Should we move this to a config file/`settings.yml`?

Reply:
My general philosophy is to only put a value in config if (1) the value is used in more than one place or (2) the value is likely to vary between environments.

Reply:
Makes sense to me.